Analyzer/plugins/Analyzer.cc

```diff
@@ -1985,52 +1864,642 @@ void Analyzer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
       // I'm sure this could be done better, if you agree and feel like it, please fix it
       // issue with a while loop and a recursive I faced is tha that mom doesnt have the same type as the genParticle
       // This is also a dupplicate code, prob should be a function
-
+      
       closestBackgroundPDGsIDs[3] = dRMinGenAndSibling;
       closestBackgroundPDGsIDs[4] = dRMinGenAndMom;
       closestBackgroundPDGsIDs[5] = fabs(genColl[closestGenIndex].pt());
       closestBackgroundPDGsIDs[6] = numSiblingsF;
     }
     // -- end TODO Sept 25
-
-    bool passPre = passPreselection(
```

Comment:
This got simplified now we pass the `passedCutsArray` only, which then can be changed, we can now have a version which is for Sept8 selection, one for the CR, one for the Ias template production